package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zwave-js/server",
-  "version": "1.10.3",
+  "version": "1.11.4",
```

Comment:
```suggestion
  "version": "1.10.4",
```
We just want to bump the patch version, right?

Reply:
doh, thank you!